host/tests/L3_TestCases/dsVideoDevice/dsVideoDevice_test2_SetAndGetZoomMode.py

```diff
@@ -78,31 +79,34 @@ def testFunction(self):
 
         result = True
 
-        self.log.testStart(self.testName, self.qcID)
-
         # Initialize the dsVideoDevice module
         self.testdsVideoDevice.initialise(self.testdsVideoDevice.getDeviceType())
 
         SupportedDevices = self.testdsVideoDevice.getSupportedVideoDevice()
         for device in SupportedDevices:
 
-            for stream in self.testStreams:
+            for stream in self.streamPaths:
                 supportedZoomModes = self.testdsVideoDevice.getSupportedZoomModes(device)
 
                 for zoomMode in supportedZoomModes:
-                    self.testPlayer.play(stream)
+                    streamPath = self.testDownloadSingleStream(stream)
+                    streamPath = streamPath.replace("\\", "/")
```

Comment:
can you re-check is this replace is required ?check other files also

Reply:
Hi @hari22yuva ,

Actually, I'm running test in windows, so in that on getting the path I received as like mentioned below,
`"/opt/karthi/dsvideodevice\\scrolling_text_fast_3840x2160_23.98fps.mp4"`

That's why I included the replace() method, then it works fine.

Reply:
Updated as we discussed. Tested and verified the same with help of JP.